### ISSUE TITLE
Phenotypes

### ISSUE THREAD

**Issue by tsalo** [bids-standard/bids-2-devel#13]
As a general (and easy) suggestion, it would be great to start collating a repository of standard phenotypic measure .json files for the commonly used Clinical/behavioural questionnaire/scales (although many are licensed and so objections may be encountered). Also, it might be worth adding some standardised clinical units (mmHg, bpm etc.,)

**Original authors**: Unknown

**Comment by tsalo** [comment]
Unless I'm misunderstanding this proposal, this seems more like an additional resource than something that belongs in the specification itself.

**Comment by yarikoptic** [comment]
yeah, and likely might relate to https://github.com/ReproNim/reproschema and its https://github.com/ReproNim/reproschema-library e.g. https://github.com/ReproNim/reproschema-library/tree/master/activities
